weldx/util.py

```diff
@@ -1278,12 +1278,18 @@ def time_ref(self, value: pd.Timestamp):
 ]
 
 
+def _array_equal(a, b):
+    if a.shape != b.shape:
+        return False
```

Comment:
interesting, does `==` not return `False` in some cases?

Reply:
It does, but the warning that I removed was actually triggered by arrays of different shape. [See here](https://stackoverflow.com/questions/44574679/python-deprecationwarning-elementwise-comparison-failed-this-will-raise-an). It would have stopped working in a future version.